Explorer/Assets/DCL/LOD/Systems/UpdateSceneLODInfoSystem.cs

```diff
@@ -72,49 +76,64 @@ private void UpdateLODLevel(ref SceneLODInfo sceneLODInfo, ref PartitionComponen
                 CheckLODLevel(ref partitionComponent, ref sceneLODInfo, sceneDefinitionComponent);
         }
 
+
+        [Query]
+        [None(typeof(DeleteEntityIntention))]
+        private void InstantiateCurrentLOD(ref SceneLODInfo sceneLODInfo, ref SceneDefinitionComponent sceneDefinitionComponent)
+        {
+            var currentLOD = sceneLODInfo.GetCurrentLOD();
+            if (!sceneLODInfo.IsDirty || currentLOD == null) return;
+
+            if (!(frameCapBudget.TrySpendBudget() && memoryBudget.TrySpendBudget())) return;
```

Comment:
Is not `frameCapBudget` redundant considering you already set budget for `AsyncInstantiateOperation`?